pigeonhole.rb

```diff
@@ -30,6 +30,14 @@
   redirect "/noise-candidates/#{today}/#{today}"
 end
 
+def search_query_input 
+  if @search
```

Comment:
Cleaner as 
return {} unless @search
{ :conditions => "and incident_key =~ /.*#{@search.strip}.*/i" }